parameters/world_modeling/world_model_base.yaml

```diff
@@ -0,0 +1,47 @@
+plugins:
```

Comment:
Improve scalability: put every plugin in a separate file. Build up world model base.yaml and include all ones there.